_services/development.md

```diff
@@ -4,15 +4,15 @@ title: Development
 slug: development
 weight: 2
 summary: |
-  Turning real-world challenges into technical solutions is among our favorite activities amidst the beakers and bunsen burners in the Savas lab. Developing novel functionality that is tailor-made for each client's needs drives our creative spirit.
+  Manifesting strategic organizational challenges via technical execution is one of the most cherished activities we participate in. We employ a wide variety of tools to tailor-make solutions for our partner's constantly evolving needs.
```

Comment:
The first sentence is pretty hard to digest and I'd recommend not ending the sentence in a preposition. I don't think that's a hard and fast rule in general writing but I'd avoid it in short, hard-hitting sections of the site like this - I think it weakens the statement.

I'd also avoid using "tailor-make;" use "tailor-made" if you can but if not change "to make custom solutions."

I'd also change "partner's" to "partners'"